### ISSUE TITLE
Enhancement: Four system improvements for Cap Vista proposal (Singapore ops, false positive filtering, scan_delta load estimation, VDES on-demand transfer)

### ISSUE THREAD

**Issue by yohei1126** [edgesentry/arktrace#26]
## Summary

Four improvements identified during pre-submission review. All are verifiable against current docs/code; none are implemented yet. Grouped into one issue for triage; can be split if scope warrants separate tracking.

---

## ① Persona 1 (Singapore) — Explicit Bunker Barge Exclusion from HDBSCAN Baseline

**Current state:** `technical-solution.md` describes HDBSCAN clustering stratified by `ship_type`, but no logic explicitly excludes small legitimate service vessels (bunker barges, water boats, pilot tenders) that operate at low SOG in Singapore waters and would otherwise score high on loitering/STS signals.

**Proposed change:** In `src/score/mpol_baseline.py`, add a pre-filter step that excludes `ship_type` values corresponding to service/bunker craft (AIS type codes 51–59, and potentially type 31/32 pilot vessels) from the HDBSCAN training set. These vessels should still be *scored* against the resulting baseline; they just should not *define* normal MPOL behavior. Document the exclusion list in `regional-playbooks.md` Persona 1 section.

**Why this matters:** Without this filter, the Singapore baseline absorbs the high-density, low-SOG stop-start patterns of bunker barges as "normal", compressing the anomaly score for genuine dark vessel STS events in the same area.

---

## ② False Positive Reduction — Promote Geopolitical Rerouting Filter to Roadmap C4

**Current state:** `regional-playbooks.md` Persona 5 (Red Sea / Middle East) documents a manual workaround: vessels rerouting via Cape of Good Hope since 2024 show legitimate AIS gaps and altered routes that trigger false positives. The workaround is analyst-applied, not automated.

**Proposed change:** Add a `--geopolitical-event-filter` option to C4 (multi-region CLI hardening) that ingests a simple YAML event file:

```yaml
events:
  - name: Red Sea Houthi conflict rerouting
    active_from: 2024-01-15
    affected_bbox: [-10, 32, 30, 80]   # Red Sea
    reroute_via_bbox: [-40, 10, -25, 40]  # Cape of Good Hope
    anomaly_weight_reduction: 0.5
```

When a vessel's route passes through a declared rerouting corridor during the active window, its AIS gap and route-deviation anomaly contributions are down-weighted by the specified factor before composite scoring. Document the Red Sea entry as a default included example.

**Why this matters:** Cape of Good Hope rerouting alone is responsible for a significant fraction of false positives in the Middle East / Indian Ocean persona. Automating the filter reduces manual triage load before physical Phase B investigations.

---

## ③ Whitelist Loop — Integrate "Cleared" Vessels from Phase B into Phase A Re-Learning

**Current state:** `roadmap.md` B6 states: *"confirmed vessels feed back into Phase A as positive labels to improve scoring."* There is no equivalent step for vessels physically inspected and cleared (outcome = `cleared`).

**Proposed change:** In the B6 Port Operations Centre dashboard integration loop, add:

1. A `cleared_vessels` table (DuckDB) populated when a Phase B investigation returns `cleared` status.
2. In `src/score/mpol_baseline.py` HDBSCAN re-training, include `cleared_vessels` MMSI as **hard negative examples** — i.e., their feature vectors anchor the "confirmed normal" region of the behavioral space.
3. In `src/score/anomaly.py` Isolation Forest retraining, add cleared vessels to the clean training set (currently seeded by `sanctions_distance ≥ 3`).

Document the data flow as a new edge in the roadmap dependency graph: `B6 (cleared) ──► A4 (negative labels)`.

**Why this matters:** Without explicit negative feedback from physical inspections, the model has no mechanism to learn from its own false positives. Cleared vessels are high-quality hard negatives because they passed ground-truth inspection.

---

## ④ VDES Bandwidth — Two-Phase On-Demand Evidence Transfer

**Current state:** `roadmap.md` B5 and `docs/field-investigation.md` describe transmitting the full evidence bundle (signed JSON + heatmap PNG + photos) in a single VDES ASM transmission. At ~307 kbps with typical evidence bundle sizes (1–5 MB per Tier 2 inspection), this blocks the VDES channel for 30–130 seconds per report.

**Proposed change:** Split B5 transmission into two phases:

**Phase B5a — Lightweight triage report (always sent immediately):**
```json
{
  "mmsi": "...",
  "tier": 2,
  "detections_summary": [...],
  "watchlist_confidence": 0.87,
  "inspector_id": "...",
  "signed_hash": "...",
  "asset_manifest": {
    "heatmap_png": {"s3_key": "...", "size_bytes": 480000, "blake3": "..."},
    "photos": [{"s3_key": "...", "size_bytes": 1200000, "blake3": "..."}]
  }
}
```

**Phase B5b — On-demand asset retrieval (shore initiates):**  
Shore station reviews the triage report, then sends a `FETCH_ASSETS` request (VDE-TDM ch 26/86) specifying which S3 keys to transmit. Patrol vessel queues the requested assets and transmits on ASM ch 24/84.

**Why this matters:** Shore analysts can often make an `escalate / clear` decision from the triage JSON alone (identity mismatch + top signals). On-demand transfer reserves scarce VDES bandwidth for cases that genuinely need high-resolution evidence, and avoids blocking the channel when multiple patrol vessels report simultaneously.

---

## Out of scope for this issue

- `scan_delta` laden/unladen algorithm specifics (belongs in `edgesentry-rs` / `trilink-core` roadmap, not this repo — see `docs/field-investigation.md` which already notes this is a `trilink-core` CP3 item)

---

## References

- `docs/technical-solution.md` — HDBSCAN, Isolation Forest, VDES channel plan
- `docs/regional-playbooks.md` — Persona 1 (Singapore), Persona 5 (Red Sea)
- `docs/roadmap.md` — B5, B6, C4
- `docs/field-investigation.md` — VDES message flow, evidence bundle format
- `src/score/mpol_baseline.py`, `src/score/anomaly.py` — scoring pipeline
